### PR TITLE
Card

### PR DESCRIPTION
**Pull request by Juantontaco**
refactor the code to match that of the player

### CODE REVIEW

**Comment by Juantontaco** [comment]
This screwed up the LavaCard default export so the feed no longer has three mock podcasts. @jrr8 take a peak at the "feed" view & determine if this way of building the card is best (compared to the previous). I felt that keeping the card & the player consistant in their build would be easier to understand down the road.

File: src/components/Card.vue [hunk 1]
```diff
@@ -1,96 +1,80 @@
...
+
+        <div class="icons d-flex">
+          <lava-player-icons :podcast="podcast"/>
+        </div>
+    </ion-card>
+  </lava-page>
 </template>
 
-
-<script lang="ts">
```
Comment (jrr8) [inline src/components/Card.vue:20]: Keep these `lang="ts"` tags. This is how we add TypeScript.

I will fix this one but in the future, keep these

Comment (jrr8) [reply src/components/Card.vue:20]: The `import Vue, { PropOptions } from 'vue'` line below and its associated usages are also TypeScript-related

**Comment by jrr8** [comment]
Once #21 gets merged we can pull -> push integration with this branch

File: src/components/Card.vue (hunk 1 above)
Comment (Juantontaco) [reply src/components/Card.vue:20]: got it and thanks for the clarification. I wasn't sure why those existed.

**Pull request merged**